### PR TITLE
Julia MNIST example

### PR DESCRIPTION
**Pull request by nathanballou**
Simple NN for MNIST dataset in Julia. Needs to wait on Julia image.

### CODE REVIEW

File: examples/julia-machine-learning/.saturn/saturn.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (jnolis) [inline examples/julia-machine-learning/.saturn/saturn.json]: It doesn't look like this example edits the `templates.json` files (either hosted or enterprise), but I think it should edit both, right?

File: examples/julia-machine-learning/.saturn/saturn.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "name": "example-julia-machine-learning",
```
Comment (jnolis) [inline examples/julia-machine-learning/.saturn/saturn.json]: I think the fact that we're training a neural network is more specific and interesting than general machine learning. So what if we rename the whole thing to reflect that

```suggestion
  "name": "example-julia-neural-network",
```

File: examples/julia-machine-learning/.saturn/saturn.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "image_uri": "public.ecr.aws/saturncloud/saturn-julia:2022.03.01`",
+  "description": "Use Julia to train a simple machine learning model.",
```
Comment (jnolis) [inline examples/julia-machine-learning/.saturn/saturn.json]:
```suggestion
  "description": "Use Julia to train a neural network model.",
```

File: examples/julia-machine-learning/MNIST-example.ipynb [hunk 4]
```diff
@@ -0,0 +1,480 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a1daeeb-2625-406d-a2f8-ebd3e9038864",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This example describes how to run a simple machine learning workflow using the famous MNIST dataset of handwritten digits. We will be labeling images of numbers. \n",
```
Comment (jnolis) [inline examples/julia-machine-learning/MNIST-example.ipynb]: Confirm my words are correct! I would say that the intro here is a little sparse so I am trying to expand it a bit. Feel free to expand it more.

```suggestion
    "This example describes how to run a neural learning workflow using the famous MNIST dataset of handwritten digits. Flux.jl is a powerful Julia library for many types of machine learning, including neural networks. We will training a neural network on images of handwritten numbers to create an image classifier. \n",
```

File: examples/julia-machine-learning/MNIST-example.ipynb [hunk 5]
```diff
@@ -0,0 +1,480 @@
...
+    "\n",
+    "## Modeling Process\n",
+    "\n",
+    "### Imports\n",
+    "This exercise uses [Flux](https://fluxml.ai/Flux.jl/stable/) to train a simple neural network. "
```
Comment (jnolis) [inline examples/julia-machine-learning/MNIST-example.ipynb]: Does this seem correct? Also is Images a base function? I don't know enough Julia...

```suggestion
    "This exercise uses [Flux](https://fluxml.ai/Flux.jl/stable/) to train a simple neural network. The code loads several functions from Flux as well as some base functions"
```

**Review by jnolis — APPROVED** [review]
Some minor changes but does not need re-review.

I would say overall though I would add more descriptive words, especially at the beginning.

**Comment by jnolis** [comment]
Also, I think this should get included on the website examples once it is ready. And maybe have it mention the fact that a different example can be used for a GPU?

**Review by jsignell — APPROVED** [review]

File: examples/julia-flux/.saturn/saturn.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "example-julia-flux",
-  "image_uri": "public.ecr.aws/saturncloud/saturn-julia-gpu:2022.03.01`",
+  "image_uri": "public.ecr.aws/saturncloud/saturnbase-julia-gpu-11.3:2022.03.01",
```
Comment (jsignell) [inline examples/julia-flux/.saturn/saturn.json]: We normally don't use the saturnbase images in examples

Comment (nathanballou) [reply examples/julia-flux/.saturn/saturn.json]: Sorry - I accidently copied the wrong one. It is fixed in the newest push.

**Pull request merged**